graphql-server/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.18 as builder
+
+WORKDIR /usr/src/graphql-schema
+COPY ./graphql-schema /usr/src/graphql-schema
+
+WORKDIR /usr/src/app
+RUN apt update && apt install -y postgresql sudo \
+	pkg-config \
+	libicu-dev
```

Comment:
do we need these? can we verify if all installed stuff are required?

Reply:
tested by removing the dependencies from the dockerfiles and then using docker-compose to start an instance with the development dockerfile,  built an image using the deployment dockerfile and started an instance via that image.